docs/topics/settings.rst

```diff
@@ -279,14 +279,23 @@ Default: ``False``
 Whether to collect verbose depth stats. If this is enabled, the number of
 requests for each depth is collected in the stats.
 
-.. setting:: DNSCACHE_ENABLED
+.. setting:: DNSCACHE_SIZE
```

Comment:
I think we should keep DNSCACHE_ENABLED option, for backwards compatibility. CachingThreadedResolver is an internal class, but DNSCACHE_ENABLED was a part of public API.

`cache_size` passed to the resolver could be 
```
cache_size = settings.getint('DNSCACHE_SIZE') if settings.getbool('DNSCACHE_ENABLED') else 0
```